### PR TITLE
[agent/node-analyzer/sysdig-deploy] Doc fixes

### PR DESCRIPTION
**Pull request by achandras**
## What this PR does / why we need it:

Various documentation fixes

## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Title of the PR starts with chart name (e.g. [mychartname])
- [x] Chart Version bumped
- [x] Variables are documented in the README.md (or README.tpl in some charts)
- [x]  Check GithubAction checks (like lint) to avoid merge-check stoppers

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/agent/README.md [hunk 1]
```diff
@@ -79,6 +79,10 @@ The following table lists the configurable parameters of the Sysdig chart and th
...
 | `resources.limits.cpu`                         | CPU limit                                                                                                                                  | ` `                                                         |
 | `resources.limits.memory`                      | Memory limit                                                                                                                               | ` `                                                         |
+| `collectorSettings.collectorHost`              | The IP address or hostname of the collector                                                                                                | ` `                                                         |
+| `collectorSettings.collectorPort`              | The port where collector is listening                                                                                                      | 6443                                                        |
```
Comment (radhikapc) [inline charts/agent/README.md]: The port number for the TCP connection of the collector service.

File: charts/sysdig-deploy/README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Currently included components:
 2. Collect the following values:
 
    - ACCESS_KEY: This is your Sysdig access key
-   - SAAS_REGION: The Sysdig SAAS region the agents will connect to. See [Regions and IP Ranges](https://docs.sysdig.com/en/docs/administration/saas-regions-and-ip-ranges/) for more information. The Collector URL is custom for on-prem installations.
+   - SAAS_REGION: The Sysdig SAAS region the agents will connect to. Possible values are `"us1"`, `"us2"`, `"us3"`, `"us4"`, `"eu1"`, `"au1"`, and `"custom"`. For on-prem installations, use `custom` and override the endpoints for each component (see [Configuration](#configuration) below).
```
Comment (radhikapc) [inline charts/sysdig-deploy/README.md]: - SAAS_REGION: The Sysdig SAAS region that the agents will connect to.  Use one of the following values:
   -  `"us1"`
   - `"us2"`
   - `"us3"`
   - `"us4"`
   - `"eu1"`, 
   - `"au1"`
   -  `"custom"`:  For on-prem installations, use `custom` and override the endpoints for each component.
 For more information, see [Configuration](#configuration).

File: charts/sysdig-deploy/README.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ Currently included components:
        --set global.clusterConfig.name=CLUSTER_NAME
     ```
 
-     **GKE Autopilot**: GKE Autopolot environments require an additional configuration parameter, `agent.gke.autopilot=true`, to install Sysdig agent:
+     **GKE Autopilot**: GKE Autopilot environments require an additional configuration parameter, `agent.gke.autopilot=true`, to install Sysdig agent:
```
Comment (radhikapc) [inline charts/sysdig-deploy/README.md]: **GKE Autopilot**: GKE Autopilot environments require an additional configuration parameter, `agent.gke.autopilot=true`, to install the Sysdig agent:

File: charts/sysdig-deploy/README.md [hunk 4]
```diff
@@ -123,6 +123,16 @@ helm repo update
 helm install -n sysdig-agent sysdig sysdig/sysdig-deploy -f values.new.yaml
 ```
 
+### Differences between `sysdig` and `sysdig-deploy`
+
+When moving from the old chart to new, there will be some differences for the agent and node-analyzer components. The majority of these differences are in metadata names and labels.
```
Comment (radhikapc) [inline charts/sysdig-deploy/README.md]: There are several differences in the agent and node-analyzer components in the new chart compared to the old one. The majority of the differences are in the metadata names and labels.

File: charts/sysdig-deploy/README.md [hunk 5]
```diff
@@ -201,15 +212,15 @@ agent:
 
 ## NodeAnalyzer
 
-For possible configuration values of the node-analyzer, please refer to the node-analyzer subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/README.md). All agent-specific configuration can be prefixed with `nodeAnalyzer.` to apply them to this chart.
+For possible configuration values of the node-analyzer, please refer to the node-analyzer subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/README.md). All specific configuration can be prefixed with `nodeAnalyzer.` to apply them to this chart.
```
Comment (radhikapc) [inline charts/sysdig-deploy/README.md]: For configuration values of the `node-analyzer`, see the node-analyzer subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/README.md).  Prefix all the specific configurations with `nodeAnalyzer.` to apply them to the chart.

File: charts/sysdig-deploy/README.md [hunk 6]
```diff
@@ -229,7 +242,7 @@ nodeAnalyzer:
 
 ## KSPM Collector
 
-For possible configuration values of the kspm-collector, please refer to the kspm-collector subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/kspm-collector/README.md). All agent-specific configuration can be prefixed with `kspmCollector.` to apply them to this chart.
+For possible configuration values of the kspm-collector, please refer to the kspm-collector subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/kspm-collector/README.md). All specific configuration can be prefixed with `kspmCollector.` to apply them to this chart.
```
Comment (radhikapc) [inline charts/sysdig-deploy/README.md]: For the configuration values of the `kspm-collector`, see the kspm-collector subchart [README](https://github.com/sysdiglabs/charts/blob/master/charts/kspm-collector/README.md). Prefix all the specific configurations with `kspmCollector.` to apply them to the chart.

**Review by radhikapc — CHANGES_REQUESTED** [review]
minor comments

File: charts/sysdig-deploy/README.md (hunk 3 above)
Comment (lilx1ao) [reply charts/sysdig-deploy/README.md]: I think the suggested change looks the same as the original one

Comment (achandras) [reply charts/sysdig-deploy/README.md]: I only noticed it after pasting, but the new text says "install **the** Sysdig agent" which sounds more right

File: charts/sysdig-deploy/CHANGELOG.md [hunk 7]
```diff
@@ -4,11 +4,21 @@
 
 This file documents all notable changes to Sysdig's sysdig-deploy Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.3.2
+
+### Minor changes
+*  Readme fixes
```
Comment (yashgiri) [inline charts/sysdig-deploy/CHANGELOG.md]:
```suggestion
* Readme fixes
* Bumped nodeAnalyzer to 1.7.2
* Bumped agent to 1.5.17
```
Just a small edit but not necessary.

**Review by yashgiri — APPROVED** [review]

File: charts/sysdig-deploy/README.md (hunk 3 above)
Comment (lilx1ao) [reply charts/sysdig-deploy/README.md]: ah my bad!

**Review by lilx1ao — APPROVED** [review]

**Pull request merged**